contrib/checkbox-ce-oem/checkbox-ce-oem-snap/series_uc24/snap/snapcraft.yaml

```
@@ -0,0 +1,139 @@
+name: checkbox-ce-oem
+summary: Checkbox CE OEM test runner and public providers
+description: "Checkbox CE OEM test runner and public providers"
+confinement: strict
+grade: stable
+
+version: '1.0-jammy'
```

This should be Noble
```suggestion
version: '1.0-noble'
```